main/src/hadd.cxx

```diff
@@ -1,39 +1,47 @@
 /**
   \file hadd.cxx
-  \brief This program will add histograms (see note) and Trees from a list of root files and write them to a target root file.
-  The target file is newly created and must not be
-  identical to one of the source files.
+  \brief This program will add histograms (see note) and Trees from a list of root files and write them to a target root
+  file. The target file is newly created and must not be identical to one of the source files.
```

Comment:
At this point shouldn't we be more specific and say that the tool allows merging (adding is also very misleading) objects from different ROOT files as long as they respect certain criteria? Also, where is the "note" this comment refers to for further explanation?

Reply:
No idea :(

Reply:
> At this point shouldn't we be more specific and say that the tool allows merging (adding is also very misleading) objects from different ROOT files as long as they respect certain criteria?

We should (the criteria is that they class has a `Merge` function)

Reply:
> Also, where is the "note" this comment refers to for further explanation?

The note is still there around line 100:
```
  \note By default histograms are added. However hadd does not support the case where
         histograms have their bit TH1::kIsAverage set.
```